src/sidebar/panel.js

```diff
@@ -37,6 +42,9 @@ function handleLocalContent(data) {
       quill.setContents(data.notes);
     }
   }
+  const contentLength = quill.getLength();
+  quill.formatText(0, contentLength, 'direction', LANG_DIR);
+  quill.formatText(0, contentLength, 'align', LANG_DIR === 'rtl' ? 'right' : 'left');
```

Comment:
this is a workaround as `setContents` with the attribute `direction` doesnt seem to work as expected. https://github.com/quilljs/quill/issues/1535